examples/one-2-one/public/index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>hasOne relation example between supplier and account</title>
```

Comment:
Please use the same description in `index.html`, `README.md` and `package.json` too. Feel free to use this description instead of what I proposed above, just make sure it's used consistently in all three places.